src/dao/V2FilterProcessor.ts

```diff
@@ -60,7 +61,11 @@ export class V2FilterProcessor implements IFilterProcessor {
       case Comparators.NOTNULL:
         return restriction.value === undefined ? V2FilterProcessor.NULL_VALUE : restriction.value;
```

Comment:
Yeah, that was preexisting code that I left alone.  Worth figuring out if ReSTv2 can handle nulls in a better way now, though.